doc/source/whatsnew/v0.17.0.txt

```diff
@@ -150,7 +150,7 @@ The Series and DataFrame ``.plot()`` method allows for customizing :ref:`plot ty
 To alleviate this issue, we have added a new, optional plotting interface, which exposes each kind of plot as a method of the ``.plot`` attribute. Instead of writing ``series.plot(kind=<kind>, ...)``, you can now also use ``series.plot.<kind>(...)``:
 
 .. ipython::
-  :verbatim:
+    :verbatim:
```

Comment:
I think the key thing is aligning it with the indentation below. This version builds properly, at least:
<img width="541" alt="screen shot 2015-09-12 at 4 31 52 pm" src="https://cloud.githubusercontent.com/assets/1217238/9834405/ce09babc-596b-11e5-8b98-f23ce1bee988.png">